api/src/main/java/marquez/api/FacetResource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018-2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.api;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.ResponseMetered;
+import com.codahale.metrics.annotation.Timed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import marquez.common.models.RunId;
+import marquez.db.OpenLineageDao;
+import marquez.service.ServiceFactory;
+import marquez.service.models.Facets;
+
+@Slf4j
+@Path("/api/v1/facets")
+public class FacetResource extends BaseResource {
+
+  private final OpenLineageDao openLineageDao;
+
+  public FacetResource(
+      @NonNull final ServiceFactory serviceFactory, @NonNull final OpenLineageDao openLineageDao) {
+    super(serviceFactory);
+    this.openLineageDao = openLineageDao;
+  }
+
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Produces(APPLICATION_JSON)
+  @Path("/run/{id}")
```

Comment:
I would add the endpoint to obtain facets for a given run to [`RunResource`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/api/RunResource.java#L31). That is, to [receive a run](https://marquezproject.github.io/marquez/openapi.html#operation/getRun), we have the endpoint:

```
GET /api/v1/jobs/runs/{id}
```

Similarly, to obtain facets for a given runs, I would follow the same pattern:

```
GET /api/v1/jobs/runs/{id}/facets
```